examples/flash_attention/example_gqa_bwd_tma_reduce_varlen.py

```
@@ -0,0 +1,792 @@
+            T.reduce_sum(acc, delta, 1)
+
+            for i in T.Parallel(blk):
+                if by * blk + i < q_current_seqlen:
+                    Delta[q_start_idx + by * blk + i, bx] = delta[i]
+
+    return flash_bwd_prep
+
+
+def make_dq_layout(dQ):
+    # bshd -> bhld to use tma reduction instruction
+    return T.Layout(dQ.shape, lambda b, l, h, d: [b, h, l, d])
```

Ruff (0.14.0)

Ambiguous variable name: `l`

(E741)